### PR TITLE
v1.4

### PR DESCRIPTION
**Pull request by francisco-dlp**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,13 +44,15 @@ requirements:
...
+    - numba
+    - numexpr
     - hyperspy-gui-ipywidgets
+    - sparse
 test:
     requires:
         - pytest
         - pytest-mpl
-        - freetype 2.8*
+        - freetype 2.9.*
```
Comment (ericpre) [inline recipe/meta.yaml]: freetype >= 2.8 instead of pinning it?

Comment (francisco-dlp) [reply recipe/meta.yaml]: It's a requirement of mpl 2.2.3. I've tested it locally and it's fine.

**Comment by ericpre** [comment]
Is it worth adding `hyperspy-gui-traitsui` as runtime dependency for convenience, now that traitsui/pyface are qt5 compatible?

**Comment by francisco-dlp** [comment]
Sounds good, I'll try.

**Comment by ericpre** [comment]
I have the same test failure locally (on linux) than the appveyor one (`test_empty_stream[True]`), I have noticed it ony recently and I haven't had time to look at it but it may be necessary now...

**Comment by ericpre** [comment]
The test failure in appveyor comes from numba=0.39! Downgrade to 0.38.1 works.

**Comment by francisco-dlp** [comment]
Good catch! If that's a numba bug it'll be good reporting it to the developers. I'll update the requirements.

**Comment by francisco-dlp** [comment]
It'll be good to also open an issue about it in the hyperspy repository to track the issue.

**Pull request merged**